paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -194,9 +194,11 @@
                 },
                 "horizontal_autoscaling": {
                     "type": "object",
+                    "additionalProperties": false,
```

Comment:
Can you confirm that every existing kubernetes.yaml file conforms to this new schema?

Reply:
Yes they are. And the old schema will still be supported until we move them to this new one.